code/datums/components/twohanded.dm

```diff
@@ -8,18 +8,18 @@
 	dupe_mode = COMPONENT_DUPE_UNIQUE_PASSARGS 		// Only one of the component can exist on an item
 	var/mob/wielder							/// The mob that is wielding us
 	var/wielded = FALSE 							/// Are we holding the two handed item properly
-	var/force_multiplier = 0						/// The multiplier applied to force when wielded, does not work with force_wielded, and force_unwielded
-	var/force_wielded = 0	 						/// The force of the item when wielded
-	var/force_unwielded = 0		 					/// The force of the item when unwielded
-	var/block_power_wielded = 0						/// The block power of the item when wielded
-	var/block_power_unwielded = 0					/// The block power of the item when unwielded
+	var/force_multiplier						    /// The multiplier applied to force when wielded, does not work with force_wielded, and force_unwielded
+	var/force_wielded 	 						    /// The force of the item when wielded
+	var/force_unwielded		 					    /// The force of the item when unwielded
+	var/block_power_wielded					     	/// The block power of the item when wielded
+	var/block_power_unwielded 				        /// The block power of the item when unwielded
```

Comment:
This should probably remain at 0, because there are things like the fireaxe which doesn't specifiy block_power_unwielded but they should be 0. Either that, or go through and find all occurances of this.